### ISSUE TITLE
2.1.4 broken for PHP 5.5

### ISSUE THREAD

**Issue by baurmatt** [wallabag/wallabag#2617]
### Issue details

2.1.4 has a composer dependency which doesn't support PHP 5.5.

### Environment

* wallabag version (or git revision) that exhibits the issue: 2.1.4
* How did you install wallabag? Via `git clone` or by downloading the package? git
* Last wallabag version that did not exhibit the issue (if applicable): 2.1.1
* php version: 5.5
* OS: Ubuntu 14.40
* type of hosting (shared or dedicated): dedicated
* which storage system you choose at install (SQLite, MySQL/MariaDB or PostgreSQL): MySQL

### Steps to reproduce/test case

* sudo -u www-data git fetch origin
* sudo -u www-data git fetch --tags
* sudo -u www-data git checkout 2.1.4 --force
* sudo -u www-data SYMFONY_ENV=prod ../bin/composer install --no-dev -o --prefer-dist

```
Loading composer repositories with package information
Installing dependencies from lock file
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for doctrine/annotations v1.3.0 -> satisfiable by doctrine/annotations[v1.3.0].
    - doctrine/annotations v1.3.0 requires php ^5.6 || ^7.0 -> your PHP version (5.5.9) does not satisfy that requirement.
  Problem 2
    - doctrine/annotations v1.3.0 requires php ^5.6 || ^7.0 -> your PHP version (5.5.9) does not satisfy that requirement.
    - jms/serializer 1.4.2 requires doctrine/annotations ^1.0 -> satisfiable by doctrine/annotations[v1.3.0].
    - Installation request for jms/serializer 1.4.2 -> satisfiable by jms/serializer[1.4.2].
```

**Comment by tcitworld** [comment]
See https://github.com/schmittjoh/serializer/pull/678

**Comment by tcitworld** [comment]
We'll need to regenerate the composer.lock on a 5.5 system.

**Comment by j0k3r** [comment]
Nope, we should only add [that](https://github.com/symfony/symfony-standard/blob/master/composer.json#L46-L50) to `composer.json` then re-generate the `composer.lock`.

**Comment by nicosomb** [comment]
Thanks to @j0k3r, this bug will be fixed and we released a 2.1.5 tomorrow.
